kolibri/plugins/coach/assets/src/views/reports/report-subheading.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+
+  <div v-if="showRecentOnly">
+    <sub v-if="standardDataTable.length">{{ $tr('subHeading', { threshold }) }}</sub>
+    <sub v-else>{{ $tr('noRecentProgress', { threshold }) }}</sub>
+  </div>
+
+</template>
+
+
+<script>
+
+  const ReportConstants = require('../../reportConstants');
+  const reportGetters = require('../../state/getters/reports');
+
+  module.exports = {
+    $trNameSpace: 'coachReportSubheading',
+    $trs: {
+      subHeading: 'Only showing activity in past {threshold} days',
+      noRecentProgress: 'No activity in past {threshold} days',
+    },
+    computed: {
+      threshold() {
+        return ReportConstants.RECENCY_THREHOLD_IN_DAYS;
```

Comment:
Dunno if we're planning on using this value anywhere else, but I think this might belong in the `state` or `pageState` more than in the constants.

I perceive constants to store strings that we're constantly referencing, but that aren't used in things like calculation. But that perception might be wrong.